lib/gakubuchi/public_exceptions.rb

```diff
@@ -0,0 +1,15 @@
+module Gakubuchi
+  class PublicExceptions < ::ActionDispatch::PublicExceptions
+    def render_html(status)
+      view_context = ::ActionView::Base.new
+      digest_path = view_context.asset_digest_path("#{status}.#{I18n.locale}.html") ||
+                    view_context.asset_digest_path("#{status}.html")
+      if digest_path
+        path = File.join(public_path, view_context.assets_prefix, digest_path)
+        render_format(status, 'text/html', File.read(path))
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->